src/clients/lan_auto.rb

```diff
@@ -141,6 +141,7 @@ def main
         end
         @new = FromAY(@param)
         Lan.Import(@new)
+        LanUdevAuto.Import(@new)
```

Comment:
well, who can say if it ever worked. Device renaming was broken in sle-11-sp3 for a very long time. In fact it was fixed before one or two months.